src/styles/getModalStyles/index.android.js

```diff
@@ -0,0 +1,169 @@
+import CONST from '../../CONST';
```

Comment:
Hmm I think we are duplicating a lot of logic here by copying this entire file. Better to just create a structure like this

```
.
└── components/
    └── Modal/
        ├── index.js - imports Modal
        ├── index.android.js - adds useNativeDriver
        └── Modal.js
```